pkgs/development/libraries/boehm-gc/default.nix

```diff
@@ -14,9 +14,6 @@ stdenv.mkDerivation rec {
     sha256 = "1798rp3mcfkgs38ynkbg2p47bq59pisrc6mn0l20pb5iczf0ssj3";
   };
 
-  buildInputs = [ libatomic_ops ];
```

Comment:
Could you verify this works on 32-bit arm? I recall having to add it for static nix compilation, but maybe that was just because boost thought it needed it:

https://github.com/NixOS/nixpkgs/pull/56281/commits/f1a5fb8051940d73f56eb5984a00705d83c4042d

Reply:
I was able to compile boehm-gc and nix for armv7l